.github/workflows/release.yml

```
@@ -22,23 +22,23 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-      
+          path: go/src/github.com/rshade/wstunnel
       - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
         with:
           go-version: '1.24'
           cache: true
-          cache-dependency-path: go.sum
+          cache-dependency-path: go/src/github.com/rshade/wstunnel/go.sum
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)